src/Api/Controllers/CiphersController.cs

```diff
@@ -630,6 +631,11 @@ public async Task PostFileForExistingAttachment(string id, string attachmentId)
                 throw new BadRequestException("Invalid content.");
             }
 
+            if (!_globalSettings.SelfHosted)
```

Comment:
@eliykat , we can't do this. Re:

> Therefore, I've also added an extra check to make sure it can't be used in cloud.

Cloud needs to support _older_ clients that don't know how to use direct uploads... we can't turn off these endpoints to cloud users yet.

Reply:
I thought the same thing, but this for uploading to attachments that already exist, not create/upload for old clients, that's line 654-698 (search for usages of `validateAttachment`).

Should be fine to do.

Reply:
Okay, cool, thanks 😁